### ISSUE TITLE
performance  so bad!

### ISSUE THREAD

**Issue by laiqun** [mangwang/PythonForFun#1]
code read all line in a file ,but there is no need to do it !

**Comment by mangwang** [comment]
@laiqun Yep, you are right. You can use the seek() function to save the memory usage when reading large files. BTW, a package name 'tailer' maybe a good choice and you can install it via pypi. Cheers!
